scapy/sendrecv.py

```diff
@@ -362,6 +379,41 @@ def sendpfast(x, pps=None, mbps=None, realtime=None, loop=0, file_cache=False, i
             raise
     finally:
         os.unlink(f)
+        return results
+
+
+def _parse_tcpreplay_result(stdout, stderr, argv, results_dict):
+    """
+    Parse the output of tcpreplay and modify the results_dict to populate output information.
+    Tested with tcpreplay v3.4.4
+    :param stdout: stdout of tcpreplay subprocess call
+    :param stderr: stderr of tcpreplay subprocess call
+    :param argv: the command used in the subprocess call
+    :param results_dict: empty dictionary to be modified when putting in results
+    :return: None
+    """
+    try:
+        stdout = str(stdout, "utf-8").replace("\t", "").split("\n")
+        stderr = str(stderr, "utf-8").replace("\t", "").split("\n")
+        print(stdout)
```

Comment:
Don't use print please. Use logging instead.

Reply:
My bad, I meant to remove that completely. No point logging if we are already returning.